release/7-2/ubuntu18.04-arm32v7/meta.json

```diff
@@ -7,8 +7,8 @@
     "Base64EncodePackageUrl": true,
     "UseAcr": true,
     "tagTemplates": [
-        "#psversion#-arm32v7-ubuntu-#shorttag#",
-        "arm32v7-ubuntu-#shorttag#"
```

Comment:
we should add a manifest list without the architecture.

Reply:
added it to the appropriate manifest lists, but keeping tag syntax to reflect processor architecture